src/main/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImpl.java

```diff
@@ -101,6 +128,16 @@ protected void processAccountingDocumentFromXml(String fileName, CreateAccountin
             }
             LOG.info("processAccountingDocumentFromXml: Finished processing accounting document XML file: " + fileName);
             
+        } catch (DuplicateCreateAccountingDocumentFileException e) {
+            reportItem.setXmlSuccessfullyLoaded(false);
+            reportItem.setNonBusinessRuleFailure(true);
+            CreateAccountingDocumentFileEntry fileEntry = e.getExistingEntry();
+            LOG.error("processAccountingDocumentFromXml: File {} was already processed by a previous run of the job; "
+                    + "it will be excluded from the current run.  File details: {}",
+                    fileEntry.getFileName(), fileEntry);

```

Comment:
Adding the fileEntry to the log statement will cause its toString() content to be added in the appropriate placeholder. I purposely overrode the BO's toString() method so that it would print the BO's properties, in case we needed that information in the logs to assist with troubleshooting.

If you think we shouldn't add that info to the logs, please let me know.

Reply:
That makes sense, and I agree, thank you